### PR TITLE
OSDOCS-2203: RN OLM upgrade to k8s 1.22

### PR DESCRIPTION
**Pull request by michaelryanpeter**
4.9 Release note

[OSDOCS-2203](https://issues.redhat.com/browse/OSDOCS-2203)

Docs Preview: https://deploy-preview-36313--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-olm-k8s-1.22

Requires: #36083

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -156,6 +156,10 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-9-olm"]
 === Operator lifecycle
 
+[id="ocp-4-9-olm-k8s-1.22"]
+==== Operator Lifecycle Manager (OLM) upgraded to Kubernetes 1.22
+
+Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM) supports Kubernetes 1.22. As a result, xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[a significant number of `v1beta1` APIs have been removed and updated to `v1`]. Before upgrading a cluster to {product-title} {product-version}, cluster administrators should xref:../operators/admin/olm-upgrading-operators.adoc[update their installed Operators to the latest channel].
```
Comment (kevinrizza) [inline release_notes/ocp-4-9-release-notes.adoc:162]: Should we include mention that if any operators that still depend on those removed apis (particularly the v1beta1 version of the CustomResourceDefinition API) are running on your cluster, they will be unable to run on 4.9?

Comment (michaelryanpeter) [reply release_notes/ocp-4-9-release-notes.adoc]: @kevinrizza PTAL and let me know if these changes work for you.

**Comment by michaelryanpeter** [comment]
@jianzhangbjz: Ready for QE review. Thank you!

**Comment by michaelryanpeter** [comment]
@jianzhangbjz: PTAL. Thanks!

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -156,6 +156,16 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-9-olm"]
 === Operator lifecycle
 
+[id="ocp-4-9-olm-k8s-1.22"]
+==== Operator Lifecycle Manager (OLM) upgraded to Kubernetes 1.22
+
+Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM) supports Kubernetes 1.22. As a result, xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[a significant number of `v1beta1` APIs have been removed and updated to `v1`]. Operators that depend on the removed `v1beta1` APIs will not run on {product-title} {product-version}. Cluster administrators should xref:../operators/admin/olm-upgrading-operators.adoc[update their installed Operators] to the `latest` channel before upgrading a cluster to {product-title} {product-version}.
```
Comment (sagidlow) [inline release_notes/ocp-4-9-release-notes.adoc:162]:
```suggestion
Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM) supports Kubernetes 1.22. As a result, xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[a significant number of `v1beta1` APIs have been removed and updated to `v1`]. Operators that depend on the removed `v1beta1` APIs will not run on {product-title} {product-version}. Cluster administrators should xref:../operators/admin/olm-upgrading-operators.adoc[upgrade their installed Operators] to the `latest` channel before upgrading a cluster to {product-title} {product-version}.
```

To maintain consistency with the linked title.

**Review by sagidlow — COMMENTED** [review]
One teeny tiny comment. Otherwise, LGTM.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (michaelryanpeter) [reply release_notes/ocp-4-9-release-notes.adoc]: Great catch! Thank you!

**Pull request merged**